algorithms/Similarity/nearest_neighbors/approximate_nearest_neighbors/tg_ann_ExprFunctions.hpp

```diff
@@ -43,7 +43,7 @@ namespace UDIMPL {
       similarity = tg_jaccard_similarity_accum(A, B);
     }
     else if (similarity_type.compare("EUCLIDEAN") == 0) {
-      similarity = tg_euclidean_similarity_accum(A, B);
+      similarity = tg_euclidean_distance_accum(A, B);
```

Comment:
Do you think the name should be consistent?  euclidean_distance make sense to me

Reply:
I think euclidean_distance is more standard. I accidentally copied it as euclidean_similarity in the first PR since all the others are similarity.